### PR TITLE
Add build and test opts to disable manual mode tests in xp-4 jobs

### PR DESCRIPTION
**Pull request by xstefank**

### CODE REVIEW

**Comment by xstefank** [comment]
@lgao, @soul2zimate, @rpelisse can anyone of you check this and merge please?

File: jobs/xp.groovy [hunk 1]
```diff
@@ -12,9 +12,29 @@ new eap7.Builder(branch:'xp-3.0.x-proposed',
 EapView.jobList(this, 'xp-3.0.x', 'xp-3.0.x.*')
 
 new eap7.Builder(branch:'xp-4.0.x',
-                 jobName: 'xp-4.0.x',
+                jobName: 'xp-4.0.x',
+                customParams: {
+                  stringParam {
+                    name("BUILD_OPTS")
```
[the remark this answers is not in the document]
Comment (xstefank) [reply jobs/xp.groovy:18]: Since this is just temporary until we have solution for docker/testcontainers I will make this change, thanks

**Review by soul2zimate — APPROVED** [review]

**Comment by xstefank** [comment]
@gaol or @soul2zimate can you merge this please?

**Comment by soul2zimate** [comment]
merged, but there is an unwanted `,` 

see https://olympus-host.lab.eng.brq.redhat.com/jenkins/job/job-configurator/246/console @xstefank Could you send a new PR for that ?

**Comment by xstefank** [comment]
thanks @soul2zimate, created https://github.com/jboss-set/cedalion/pull/50

**Pull request merged**